lib/Scraped/json.rb

```diff
@@ -0,0 +1,23 @@
+require 'scraped'
+
+module Scraped
+  class JSON < Scraped::Document
+    require 'json'
+
+    private
+
+    def initialize(json: nil, **args)
+      super(**args)
+      @json = json
+    end
+
+    def json
+      @json ||= ::JSON.parse(Nokogiri::HTML(response.body), symbolize_names: true)
+    end
+
+    def fragment(mapping)
+      noko_fragment, klass = mapping.to_a.first
```

Comment:
`noko_fragment` is a rather odd name here, when this is json, and nothing to do with nokogiri. If this is to deliberately highlight that it's lifted from elsewhere, then I suspect you're better either removing the need for the duplication, though that might be best happening upstream (as it'll probably involve making this and `Scraped::HTML` have a common base class), or leaving a comment behind to explain to later readers what's going on.